app/assets/stylesheets/ltags/GithubTag.scss

```diff
@@ -337,22 +335,22 @@
 .gh-btn-container {
   text-align: center;
   padding: 0.35em 0 1.35em;
-  box-shadow: -0px -0px 60px 42px #fff;
-  box-shadow: -0px -0px 60px 42px var(--card-bg);
+  box-shadow: -0 -0 60px 42px var(--card-bg, #fff);
   position: relative;
   z-index: 100;
 }
 
 .gh-btn {
-  color: #0366d6;
-  background-color: #f1f8ff;
+  color: var(--accent-brand);
+  background-color: var(--base-100);
```

Comment:
(blocking) - this is giving the button near-black background in light themes, and unfortunately the contrast isn't strong enough for it to be accessible:

<img width="744" alt="screenshot showing button with black background" src="https://user-images.githubusercontent.com/20773163/109522717-7752b180-7aa6-11eb-9d85-d7ee31685911.png">

I think in this case we may need to revert to the previous hard-coded colours (CCing @ludwiczakpawel for some design eyes on this)

Reply:
yeah, I've been experimenting with different css variables but most of them change based on the theme being used. 
I thinking of revert to the original hard-coded white color, but I still get a low contrast (3.36). What do you think would be the best approach?

Reply:
Good catch! I'm going to tag our product team so they can weigh in, since the CSS changes will affect the look of these liquid tags in different themes. I know our designers have been looking at liquid tags recently and might have a view on how we approach this at the moment 👍  CC @forem/product

Reply:
I can see two options right now:

1. Since you're changing the markup of this liquid tag anyway, maybe we could replace that "hardcoded" button with something like
    ```
    <a href="..." class="crayons-btn crayons-btn--secondary"> ...
    ```
    Thanks to this, it will now rely on our [Design System buttons](https://storybook.forem.com/?path=/story/components-buttons--default) so it should have good-enough contrast ratio AND work properly with other themes.
2. Alternative would be to just stick with the original, hardcoded colors (and maybe adjust them a little to have better contrast) even though they were not coming from our design system.

If option 1 is too much or too complex we should go with option 2 especially we will need to look into liquid tags more holistically at some point anyway...

Reply:
@aitchiss @ludwiczakpawel Cool works fine! I've updated the description with the new screenshots